File: app/src/main/java/com/masterjefferson/flockulator/onboard/welcome/WelcomeView.java
package com.masterjefferson.flockulator.onboard.welcome;

/**
 * ${FILE_NAME}
 * Created by jeff on 9/21/17.
 */

public interface WelcomeView {

  void openAuthorizationPage(String url);
  void setLoading(boolean loading);
  void proceedToApplication();
}

File: app/src/main/java/com/masterjefferson/flockulator/onboard/welcome/WelcomeView.kt
package com.masterjefferson.flockulator.onboard.welcome

/**
 * ${FILE_NAME}
 * Created by jeff on 9/21/17.
 */

interface WelcomeView {

    fun openAuthorizationPage(url: String)
    fun setLoading(loading: Boolean)
    fun proceedToApplication()
}
